import { AppError } from "./AppError";

export class NotFoundError extends AppError {
  constructor(message: string = "Recurso n√£o encontrado") {
    super(message, 404);
  }
}